admin/client/components/AlertMessages.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Alert } from 'elemental';
+
+/**
+ * This renders alerts for API success and error responses.
+ *   Error format: {
+ *     error: 'validation errors' // The unique error type identifier
+ *     detail: { ... } // Optional details specific to that error type
+ *   }
+ *   Success format: {
```

Comment:
I'm not sure the success format makes as much sense because the API doesn't return data in that format. The error format is solid though so I figured I'd make the component handle success and error in the same format for now to keep consistency. It might make sense to just pass in a string for success cases.